deploy/helm/sumologic/values.yaml

```diff
@@ -2821,12 +2834,46 @@ otelcol:
                 from_attribute: k8s_container_name
               - action: delete
                 key: k8s_container_name
-          groupbyattrs:
+          attributes/remove_fluent_tag:
+            actions:
+              - action: delete
+                key: fluent.tag
+
+          groupbyattrs/fluent_tag:
+            keys:
+              - fluent.tag
+          groupbyattrs/containers:
             keys:
               - k8s.container.id
               - k8s.container.name
               - k8s.namespace.name
               - k8s.pod.name
+          groupbyattrs/systemd:
+            keys:
+              - _SYSTEMD_UNIT
+              - _HOSTNAME
+
+          filter/include_fluent_tag_containers:
+            logs:
+              include:
+                match_type: regexp
+                record_attributes:
+                  - key: fluent.tag
+                    value: containers\..+
+          filter/include_fluent_tag_host:
+            logs:
+              include:
+                match_type: regexp
+                record_attributes:
+                  - key: fluent.tag
+                    value: host\..+
+          filter/include_systemd:
+            logs:
+              include:
+                match_type: regexp
+                record_attributes:
+                  - key: _SYSTEMD_UNIT
+                    value: .+
```

Comment:
It should exclude kubelet:

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/79a0ea746d53f4f5f4bc3463a0a0c4b8ac299401/deploy/helm/sumologic/conf/logs/fluentd/logs.source.systemd.conf#L5-L8